src/common/aarch64/cpu_detect.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright 2013 Dolphin Emulator Project / 2021 Citra Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <cstring>
+#include <fstream>
+#include <sstream>
+#include <string>
+
+#if !defined(_WIN32) && !defined(__APPLE__)
+#ifndef __FreeBSD__
+#include <asm/hwcap.h>
+#endif
+#include <sys/auxv.h>
+#include <unistd.h>
+#endif
+
+#include "common/aarch64/cpu_detect.h"
+#include "common/common_types.h"
+#include "common/file_util.h"
+namespace Common {
+#ifndef WIN32
+const char procfile[] = "/proc/cpuinfo";
```

Comment:
```suggestion
constexpr char procfile[] = "/proc/cpuinfo";
```
and move this inside `GetCPUString()`